### ISSUE TITLE
What happened to all the code?

### ISSUE THREAD

**Issue by jfly** [nextstepwebs/simplemde-markdown-editor#1]
Did this project move to https://github.com/sparksuite/simplemde-markdown-editor? It would be nice to add a README explaining the move.

**Comment by alexvikman** [comment]
I don't think this was a move, but rather someone that hijacked their Github account.

**Comment by battlesquid** [comment]
Well, someone should update the pages, I got super confused when I saw that the HTML was a piece of crap :P

**Comment by gclove** [comment]
What's wrong ?  😭

**Comment by doxiaodong** [comment]
interesting

**Comment by mdschweda** [comment]
Ugh... 😒

**Comment by adampatterson** [comment]
I'm not 100% sure what happened, but I was looking at the source for a WordPress plugin.

I found this https://github.com/sparksuite/simplemde-markdown-editor which is probably the original repo.
